projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.ts

```diff
@@ -1,41 +1,35 @@
 import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
 import {
   CmsBreadcrumbsComponent,
-  isNotNullable,
   PageMetaService,
   TranslationService,
 } from '@spartacus/core';
 import { combineLatest, Observable } from 'rxjs';
-import { filter, map } from 'rxjs/operators';
+import { map } from 'rxjs/operators';
 import { CmsComponentData } from '../../../cms-structure/page/model/cms-component-data';
+import { PageTitleComponent } from '../page-header/page-title.component';
 
 @Component({
   selector: 'cx-breadcrumb',
   templateUrl: './breadcrumb.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class BreadcrumbComponent implements OnInit {
-  title$: Observable<string>;
+export class BreadcrumbComponent extends PageTitleComponent implements OnInit {
```

Comment:
Why is the breadcrumbComponent extending pageTitleComponent now?

Reply:
@hackergil because we created component which uses part of logic from breadcrumb, we moved this part of logic to new, smaller component and now there is no code duplication. This solution doesn't provide side effects.